nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -379,15 +378,16 @@ async def purge(cls, driver: Driver, kbid: str):
             shards_obj = writer_pb2.Shards()
             shards_obj.ParseFromString(payload)  # type: ignore
 
-            if cluster_settings.standalone_mode:
-                await load_active_nodes()
+            await load_active_nodes()
```

Comment:
This is. the actual fix: turns out that purge command does not run with the cluster monitor sidecar container, thus it will not be part of the chitchat cluster.